endgame_models/__init__.py

```diff
@@ -4,10 +4,11 @@
     apply_incremental_param_changes,
     make_endgame_model,
 )
-
+from .get_differences import get_read_only_differences
 __all__ = [
```

Comment:
`__all__` on top pls. Although I'd probably not put it here at all, a init just contians imports here.

Reply:
Done - I've left it in the init purely because it shows the inits as being in use when it's present (not greyed out)